From 683890c83c2114172ecb8cafb06eea437d0b89cf Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 22 Feb 2020 23:05:23 +0200
Subject: [PATCH 3/4] remove up+down network icon shows

	modified:   sdk/src/java/org/lineageos/internal/statusbar/NetworkTraffic.java
---
 .../java/org/lineageos/internal/statusbar/NetworkTraffic.java | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/sdk/src/java/org/lineageos/internal/statusbar/NetworkTraffic.java b/sdk/src/java/org/lineageos/internal/statusbar/NetworkTraffic.java
index d42484c7..6082e235 100644
--- a/sdk/src/java/org/lineageos/internal/statusbar/NetworkTraffic.java
+++ b/sdk/src/java/org/lineageos/internal/statusbar/NetworkTraffic.java
@@ -465,9 +465,7 @@ public class NetworkTraffic extends TextView {
 
     private void updateTrafficDrawable() {
         final int drawableResId;
-        if (mMode == MODE_UPSTREAM_AND_DOWNSTREAM) {
-            drawableResId = R.drawable.stat_sys_network_traffic_updown;
-        } else if (mMode == MODE_UPSTREAM_ONLY) {
+        if (mMode == MODE_UPSTREAM_ONLY) {
             drawableResId = R.drawable.stat_sys_network_traffic_up;
         } else if (mMode == MODE_DOWNSTREAM_ONLY) {
             drawableResId = R.drawable.stat_sys_network_traffic_down;
-- 
2.17.1

